helm-chart/renku-notebooks/values.yaml

```diff
@@ -40,26 +42,30 @@ securityContext:
 # turns on flask debug mode
 debug: true
 
+## Default server options - these will be provided to the user in the UI
```

Comment:
I think this should be in a separate PR.

Reply:
hmm... just adding a few lines of documentation to the values file... but ok.

Reply:
Sorry, I thought that had ended up in here by accident because it has nothing to do with the title/description of the PR or the commit message. I'm perfectly fine with having "non-atomic" PRs, but at least having two commits for things that have nothing in common would indicate that this was intended.

Reply:
no worries, I already have a new PR for that, can include more documentation (and fix the README) there as well